src/media/mod.rs

```diff
@@ -525,6 +525,27 @@ impl Media {
             remote_pts: a.pts,
             remote_exts: a.exts,
             remote_created: false,
+            simulcast: if let Some(simulcast) = a.simulcast {
+                Some(SdpSimulcast {
+                    send: crate::sdp::SimulcastGroups(
+                        simulcast
+                            .send
+                            .into_iter()
+                            .map(|r| crate::sdp::RestrictionId::new_active(r.to_string()))
+                            .collect(),
+                    ),
+                    recv: crate::sdp::SimulcastGroups(
+                        simulcast
+                            .recv
+                            .into_iter()
+                            .map(|r| crate::sdp::RestrictionId::new_active(r.to_string()))
+                            .collect(),
+                    ),
+                    is_munged: false,
+                })
+            } else {
+                None
+            },
```

Comment:
Refactor this so that you can write `a.simulcast.map(|s| s.into_sdp()` and move the actual code body elsewhere.

Reply:
Done